### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Dmytro-Kretov**
- [DEMO LINK](https://Dmytro-Kretov.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Dmytro-Kretov.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by marina-tilniak — CHANGES_REQUESTED** [review]
картки не мають ставати ширшими на велих екранах
image

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,278 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
```
Comment (vkryvytskyy) [inline src/index.html:10]: image
Did you deploy your work with `npm run deploy` after you've made changes to your code?

**Review by vkryvytskyy — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Dmytro-Kretov) [reply src/index.html:10]: Yes, and I did it one more time

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,209 @@
...
+  border-radius: 8px;
+}
+/*#endregion header*/
+
+/*#region Card*/
+.card {
+  display: flex;
+  flex-direction: column;
+  margin-right: 50px;
+  flex: 0 0 19%;
```
Comment (Bohdan-Maksymiuk) [inline src/styles/main.scss:77]: .

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Previous comment about card stretching didn’t fixed
and you can separate styles into separate files according to BEM blocks

**Comment by Dmytro-Kretov** [comment]
hope this time is fine. 
if smth is wrong, please give a hint on where to make changes

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**